include/dmlc/logging.h

```diff
@@ -14,6 +14,10 @@
 #include <stdexcept>
 #include "./base.h"
 
+#if DMLC_LOG_STACK_TRACE
+#include <execinfo.h>
```

Comment:
MinGW (at least the versions 4.9.3 and 6.2.0 that I've got on my machine) doesn't seem to have execinfo.h